prisma/seed/voyage.ts

```diff
@@ -118,4 +118,52 @@ export const populateVoyages = async () => {
             showcasePublishDate: new Date("2025-03-03T06:59:59.000Z"),
         },
     });
+
+    await prisma.voyage.create({
+        data: {
+            number: "54",
+            status: {
+                connect: {
+                    name: "Upcoming",
+                },
+            },
+            startDate: new Date("2025-03-03T17:00:00.000Z"),
+            endDate: new Date("2025-04-13T04:59:59.000Z"),
```

Comment:
I think endDate should be `new Date("2025-04-14T04:59:59.000Z")`
since all these are in UTC, and the voyage dates /times on discord are in Jim's time, start time would be the same day for both timezones. 

In short, these need to be converted to UTC/GMT+0

Please see https://github.com/chingu-x/chingu-dashboard-be/wiki/Timezone